### PR TITLE
Add consensus version

### PR DESCRIPTION
**Pull request by kvinwang**

### CODE REVIEW

File: pruntime_rpc.proto [hunk 1]
```diff
@@ -136,6 +136,10 @@ message SystemInfo {
   uint64 number_of_clusters = 5;
   // The number of contracts deployed
   uint64 number_of_contracts = 6;
+  // The consensus version used to coordinate the pruntime's behavior
+  uint32 consensus_version = 7;
```
Comment (h4x3rotab) [inline pruntime_rpc.proto:140]: Does it mean the current activated version on the blockchain, while the "max" means the version supported by this binary?

Comment (kvinwang) [reply pruntime_rpc.proto:140]: Yep.

Comment (h4x3rotab) [reply pruntime_rpc.proto:140]: Would love to add the explanation to the comment

**Comment by kvinwang** [comment]
Merged

**Pull request closed without merging**